src/fides/api/util/consent_util.py

```diff
@@ -587,31 +581,14 @@ def upsert_default_experience_config(
         db=db, object_id=experience_config_schema.id
     )
 
-    if existing_experience_config:
+    if not existing_experience_config:
         logger.info(
-            "Checking default experience config {} for updates",
-            existing_experience_config.id,
-        )
-
-        dry_update: PrivacyExperienceConfig = existing_experience_config.dry_update(
-            data=experience_config_schema.dict(exclude_unset=True)
+            "Creating default experience config {}", experience_config_schema.id
         )
-        # Validating some required fields if this is an overlay
-        ExperienceConfigCreate.from_orm(dry_update)
-
-        update_data = experience_config_schema.dict(exclude_unset=True)
-        del update_data["component"]
-        del update_data["id"]
-        # This is important for making sure config is only updated if it actually changed
-        existing_experience_config.update(
-            db=db, data=ExperienceConfigUpdate(**update_data).dict(exclude_unset=True)
+        return PrivacyExperienceConfig.create(
+            db,
+            data=experience_config_schema.dict(exclude_unset=True),
+            check_name=False,
         )
-        return False, existing_experience_config
 
-    logger.info("Creating default experience config {}", experience_config_schema.id)
-    new_experience_config = PrivacyExperienceConfig.create(
-        db,
-        data=experience_config_schema.dict(exclude_unset=True),
-        check_name=False,
-    )
-    return True, new_experience_config
+    return None
```

Comment:
nit: we _could_ keep a debug/info log statement for this scenario, i.e. when it's not detected as a new experience config, just to help debug when an experience config is unexpectedly not getting created? may be overkill...

Reply:
good idea thank you!

Reply:
addressed in [51ba914](https://github.com/ethyca/fides/pull/3605/commits/51ba914acad1399dcc3ab3e18809f11c1f8d0b24)